src/components/Tags/docs/index.story.tsx

```diff
@@ -171,6 +171,20 @@ export default {
                 ],
                 fonts: [],
```

Comment:
Please add a control to change fonts

Reply:
i also added a font size definition
it drive me crazy seeing the size increase to 30px